style.css

```diff
@@ -1,17 +1,33 @@
+:root {
+  --dark-red: #dd0303;
+  --red: #ff0000;
+  --dark-gray: #2f363e;
+  --light-gray: #f7f7f7;
+  --gray: #f0f0f0;
+  --white: #fff;
+  --black: #000;
+  --shadow: rgba(0, 0, 0, 0.1);
+}
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: "Raleway", "Roboto", "Open Sans", sans-serif;
```

Comment:
Do you need to use universal selector to apply a general font rule?

Would it be more usual to apply this to the body? Have a think.